bigchaindb/web/views/outputs.py

```diff
@@ -24,5 +24,5 @@ def get(self):
         with pool() as bigchain:
             outputs = bigchain.get_outputs_filtered(args['public_key'],
                                                     include_spent)
-            # NOTE: We pass '..' as a path to create a valid relative URI
-            return [u.to_uri('..') for u in outputs]
```

Comment:
It is still being used for instance to check that all the inputs in a transaction are unique. I guess the same could be achieve without it since the `TransactionLink` implements `__hash__` but that requires some refactor that was not in the scope of this pr